app/routes/send-endringsmelding.tsx

```diff
@@ -2,119 +2,142 @@ import HovedInnhold from '~/komponenter/hovedInnhold/HovedInnhold';
 import TekstBlokk from '~/komponenter/tekstblokk/TekstBlokk';
 import StegIndikator from '~/komponenter/stegindikator/StegIndikator';
 import React, { useEffect, useState } from 'react';
-import { useSpråk, useTekster } from '~/hooks/contextHooks';
+import {
+  useEndringsmeldingMottattDato,
+  useSpråk,
+  useTekster,
+} from '~/hooks/contextHooks';
 import { Button, Textarea } from '@navikt/ds-react';
-import { useNavigate } from '@remix-run/react';
+import { Form, useActionData, useNavigate, useSubmit } from '@remix-run/react';
 import Veiledning from '~/komponenter/veiledning/Veiledning';
 import css from './send-endringsmelding.module.css';
 import { ETypografiTyper } from '~/typer/typografi';
 import { ESanityMappe, ESteg } from '~/typer/felles';
 import { hentPathForSteg } from '~/utils/hentPathForSteg';
+import { sendEndringsmelding } from '~/utils/sendEndringsmelding';
+import { ActionArgs } from '@remix-run/node';
+import { EFritekstFeil } from '~/typer/fritekstfeil';
+import {
+  MAKS_INPUT_LENGDE,
+  RESPONSE_STATUS_OK,
+  SPESIAL_TEGN_REGEX,
+} from '~/konstanter/sendEndringsmelding';
+import { getSession } from '~/sessions';
+import { i18nInnhold } from '~/utils/i18n';
+
+export async function action({ request }: ActionArgs) {
+  const formData = await request.formData();
+  const endringsmelding = formData.get('endringsmelding') as string;
+  return await sendEndringsmelding(
+    endringsmelding,
+    await getSession(request.headers.get('Cookie')),
+  ).then(response => {
+    if (!response.ok) {
+      console.log('Feil i send endringsmelding');
+      throw Error('Det skjedde en feil under POST til backend');
+    }
+    return response;
+  });
+}
 
 export default function SendEndringsmelding() {
+  const actionData = useActionData<typeof action>();
+  const submit = useSubmit();
+
   const tekster = useTekster(ESanityMappe.SEND_ENDRINGER);
   const teksterFelles = useTekster(ESanityMappe.FELLES);
-
   const navigate = useNavigate();
   const [språk] = useSpråk();
+  const [, settEndringsmeldingMottattDato] = useEndringsmeldingMottattDato();
 
-  const spesialTegnRegex = /[!@#$%^&*()?"{}|<>+¨=]/;
-  const MAKS_INPUT_LENGDE = 1000;
+  const [valideringsfeil, settValideringsfeil] = useState<EFritekstFeil | null>(
+    EFritekstFeil.MANGLER_TEKST,
+  );
```

Comment:
Forslag (kan lages som egen oppgave): enten ha en liste med feilmeldinger eller ha to states for feilmeldinger (siden det kun er to som kan vises samtidig)

Foreløpig går det fint å kun vise 1

Reply:
Okei da viser vi en feilmelding for tekstfeltet for nå. 
Ny trello oppgave her: https://trello.com/c/p735NNeP/111-flere-feilmeldinger